packages/driver/src/cy/commands/request.coffee

```diff
@@ -125,6 +125,15 @@ module.exports = (Commands, Cypress, cy, state, config) ->
       if originOrBase = config("baseUrl") or cy.getRemoteLocation("origin")
         options.url = $Location.qualifyWithBaseUrl(originOrBase, options.url)
 
+      ## Make sure the url unicode characters are properly escaped
+      ## https://github.com/cypress-io/cypress/issues/5274
+      try
+        options.url = decodeURI(options.url)
+      catch URIError
+        # The url is not already encoded just silence the error
```

Comment:
Can you give me an example of when this will throw a `URIError`? I tried some different inputs but couldn't get it to throw.

Reply:
An url with percents value in it like: `http://localhost:1234/%` will trigger this error.

Since the decodeURI will interpret the littéral percent character as the begin of encoding sequence.